### ISSUE TITLE
Question regarding to Conv2D

### ISSUE THREAD

**Issue by sudonto** [noureldien/timeception#4]
Hi @noureldien , nice work. In your paper, you put Conv2D in Temporal Conv Module. But, after plotting your model to jpeg, I did not see the presence of Conv2D layer in that module. 

![image](https://user-images.githubusercontent.com/34728756/59995479-3465f700-9692-11e9-95f4-6dc1ef4e9a53.png)

In my graph, it's not Conv2D but BatchNorm layer. Can you explain more about this? Thanks.

**Comment by noureldien** [comment]
Hi, thanks for asking. Conv2D here means 1x1x1 Conv3D dimension reduction. You find it here:

Keras/Tensorflow
https://github.com/noureldien/timeception/blob/master/nets/timeception.py#L120
Either you do depth-wise conv (Cov1D) then dimension reduction (1x1x1 Conv3D), or you do first dimension reduction (1x1 Conv2D) followed by depth-wise conv (Cov1D). Both will get you the same result.
